src/NSwag.SwaggerGeneration.AspNetCore/AspNetCoreToSwaggerGenerator.cs

```diff
@@ -146,14 +146,31 @@ public async Task<SwaggerDocument> GenerateAsync(object serviceProvider)
                     }
                 };
 
-                operations.Add(new Tuple<SwaggerOperationDescription, ApiDescription, MethodInfo>(operationDescription, apiDescription, method));
+                var consumes = apiDescription.SupportedRequestFormats
+                   .Select(f => f.MediaType).Concat(
+                       apiDescription.ActionDescriptor.ActionConstraints.OfType<ConsumesAttribute>().SelectMany(a => a.ContentTypes))
```

Comment:
@pranavkm Any idea why the ConsumesAttribute is not reported in SupportedRequestFormats by api explorer and I have to manually search for the attributes?

Reply:
Very nice. Thank you very much.

Reply:
Pranav, thanks for clarifying, PR updated with custom input formatter